guides/common/modules/proc_registering-hosts-for-conversion.adoc

```diff
@@ -10,7 +10,6 @@ Activation keys produced by the Ansible role are as follows:
 * convert2rhel_oracle7 - used for registering oracle7 hosts.
 * convert2rhel_oracle8 - used for registering oracle8 hosts.
 * convert2rhel_rhel7 - used for re-registering a host converted to RHEL 7.
-* convert2rhel_rhel8 - used for re-registering a host converted to RHEL 8.
```

Comment:
Are you absolutely certain this is removed?

Reply:
No. That was unintentional. Putting it back.